py_trees_ros/action_clients.py

```diff
@@ -216,7 +216,7 @@ def update(self):
             # goal was rejected
             self.feedback_message = "goal rejected"
             return py_trees.common.Status.FAILURE
-        if self.result_status is None:
+        if self.result_status is None or not self.get_result_future:
```

Comment:
For consistency re: use of "truthiness" in these conditions, could you do:

```suggestion
        if self.result_status is None or self.get_result_future is None:

Reply:
@sea-bass done!